examples/lighting-app/efr32/BUILD.gn

```diff
@@ -23,6 +23,7 @@ assert(current_os == "freertos")
 efr32_project_dir = "${chip_root}/examples/lighting-app/efr32"
 examples_plat_dir = "${chip_root}/examples/platform/efr32"
 
+chip_print_memory_usage = false
```

Comment:
To be able to override variables on the command line, they must be defined inside `declare_args()`.

```
declare_args() {
  # Dump memory usage at link time.
  chip_print_memory_usage = false
}
```

Reply:
Thank you Michael. I updated the files